.github/workflows/clippy.yml

```diff
@@ -1,6 +1,8 @@
 on:
   push:
   pull_request:
+    branches-ignore:
+      - 'master'
```

Comment:
i think this needs to sit [under push and not pull_request](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#onpushpull_requestbranchestags)

Reply:
I'm an idiot. Will fix.